src/did/did.spec.ts

```diff
@@ -573,14 +576,17 @@ describe('DID Resolver', function test() {
       const controllerDid = await runtimes[0].did.convertIdentityToDid(ownerIdentity);
       const controllerDidDoc = await runtimes[0].did.getDidDocument(controllerDid);
       const did = await runtimes[0].did.convertIdentityToDid(aliasIdentity);
-      const authKeyIds = controllerDidDoc.publicKey.map((key) => key.id).join(',');
+      const authKeyIds = [];
+      for (const entry of controllerDidDoc.publicKey) {
+        authKeyIds.push(entry.id);
+      }
 
       const expectedDefaultDid = {
         '@context': 'https://w3id.org/did/v1',
         id: did,
         controller: controllerDidDoc.id,
         authentication: [
-          authKeyIds,
+          authKeyIds.join(','),
```

Comment:
Wouldn't this mean, that we just add
_one_ authkey, like `"did1#key1,did1#key2,did1#key3"`
instead of _multiple_ keys `["did1#key1", "did1#key2", "did1#key3"]`?